Exercise-13/js/index.js

```diff
@@ -0,0 +1,39 @@
+import data from "../assets/friends.json" assert { type: "json" };
+
+const cardsContainer = document.getElementById("cardsContainer");
+
+function addFriendsData() {
```

Comment:
could you also add comments to functions

Reply:
ok, I will do that